### DISCUSSION TITLE
[Feature] <dates and times in Albums>

### DISCUSSION THREAD

**Discussion by nickdataskill** [immich-app/immich#10980] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

You guys have copied one annoyance that I have from the-app-that-shall-not-be-named...
Albums in web view show dates and times (excellent) however in the Android App they don't - I'd love for this to be added as some of my albums cover large periods of time

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4739
